lib/fat_free_crm/gem_dependencies.rb

```diff
@@ -5,6 +5,7 @@
 #------------------------------------------------------------------------------
 require 'rails/all'
 require 'rails-observers'
+require 'responders'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.